https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=235301

setuptools_scm uses setuptools_scm for it's build and it does not build
with old versions of setuptools_scm installed.

--- setuptools_scm-3.2.0/setup.py	2019-01-16 20:34:04.000000000 +0000
+++ setuptools_scm-3.2.0/setup.py	2019-04-15 06:53:20.259350422 +0000
@@ -22,7 +22,7 @@ def scm_config():
     has_entrypoints = os.path.isdir(egg_info)
     import pkg_resources
 
-    sys.path.insert(0, src)
+    sys.path.insert(0, here)
     pkg_resources.working_set.add_entry(src)
     from setuptools_scm.hacks import parse_pkginfo
     from setuptools_scm.git import parse as parse_git
